python/hail/expr/expressions/typed_expressions.py

```diff
@@ -2465,6 +2465,29 @@ def position(self):
         """
         return self._field("position", tint32)
 
+    def global_position(self):
+        """Returns a global position, computed as the sum of the local position
+        and the chromosome's starting position on this locus's reference genome.
+        See also :meth:`.locus_from_global_position`.
+
+        Examples
+        --------
+
+        >>> hl.locus('3', 100).global_position().value
+        492450093
+
+        >>> hl.locus('chr3', 100, 'GRCh38').global_position().value
+        491150050
+
+        Returns
+        -------
+        :class:`.Expression` of type :py:data:`.tint64`
+            Global base position of locus along the reference genome.
+        """
+        reference_genome = self.dtype.reference_genome
+        return construct_expr(ApplyMethod('locusToGlobalPos({})'.format(reference_genome), self._ast), 
+                              tlocus(reference_genome), self._indices, self._aggregations)
```

Comment:
this type is wrong. This makes me think we need Python tests that would catch it.